packages/marqus-desktop/src/renderer/components/Monaco.tsx

```diff
@@ -149,6 +145,17 @@ export function Monaco(props: MonacoProps): JSX.Element {
 
   // Mount / Unmount
   useEffect(() => {
+    // dragenter and dragover have to be cancelled in order for the drop event
+    // to work on a div.
+    const dragEnter = (ev: DragEvent) => {
+      ev.stopPropagation();
+      ev.preventDefault();
+    };
+    const dragOver = (ev: DragEvent) => {
+      ev.stopPropagation();
+      ev.preventDefault();
+    };
```

Comment:
These were pulled into the `useEffect` hook to avoid regenerating them each render.